packages/mds-utils/validators.ts

```diff
@@ -161,14 +161,19 @@ export const isValidAuditIssueCode = (
 export const isValidAuditNote = (note: unknown, options: Partial<ValidatorOptions> = {}): note is string =>
   Validate('note', note, auditNoteSchema, options)
 
-export const isStringifiedTelemetry = (telemetry: any): telemetry is StringifiedTelemetry => {
-  return !!telemetry.gps
+export const isStringifiedTelemetry = (telemetry: unknown): telemetry is StringifiedTelemetry => {
+  return !!(telemetry as StringifiedTelemetry).gps
 }
 
-export const isStringifiedEventWithTelemetry = (event: any): event is StringifiedEventWithTelemetry => {
-  return event.event_type && event.telemetry
+export const isStringifiedEventWithTelemetry = (event: unknown): event is StringifiedEventWithTelemetry => {
+  return !!((event as StringifiedEventWithTelemetry).event_type && (event as StringifiedEventWithTelemetry).telemetry)
 }
 
-export const isStringifiedCacheReadDeviceResult = (device: any): device is StringifiedCacheReadDeviceResult => {
-  return device.device_id && device.provider_id && device.type && device.propulsion
+export const isStringifiedCacheReadDeviceResult = (device: unknown): device is StringifiedCacheReadDeviceResult => {
+  return !!(
+    (device as StringifiedCacheReadDeviceResult).device_id &&
```

Comment:
is there a cleaner way to do this?  4x casts seems .... suboptimal?

Reply:
Type assertions inside TypeScript type guards are pretty commonplace. There is no runtime performance impact, but it does make it difficult to read in this instance. I'll see if there's a cleaner way.
 
> The reason why it's not called "type casting" is that casting generally implies some sort of runtime support. However, [type assertions](https://basarat.gitbooks.io/typescript/docs/types/type-assertion.html) are purely a compile time construct and a way for you to provide hints to the compiler on how you want your code to be analyzed.